test/NCI.OCPL.Api.Glossary.Tests/Tests/Controllers/TermsControllerTest.cs

```diff
@@ -126,72 +128,232 @@ public async void GetById()
         }
 
         [Fact]
-        public async void GetById_BlankRequiredFields()
```

Comment:
A test for required items not being set was removed.  Was this deliberate? (Totally cool if it was a bad test.)

Reply:
The optional parameter requestedFields is no longer passed into GetById, and the unit test to check whether just the required fields were being passed in correctly was still in the code (I had forgotten to delete it last time). I removed it since it's no longer necessary to test!